README.md

```diff
@@ -1 +1,10 @@
-# docker-nexus3-crowd
\ No newline at end of file
+# docker-nexus3-crowd
+
+Docker image that installs [nexus3-crowd-plugin](https://github.com/pingunaut/nexus3-crowd-plugin)
+
+At runtime, the following environment variables need to be passed to the container
+- CROWD_URL
+- CROWD_USER
+- CROWD_PASSWORD
```

Comment:
This means anyone with access to the ECS task definition or task can see the password. IMO it would be better to pass it an S3 URL and load this config file from S3.